app/src/main/java/com/cornellappdev/score/components/ScoreSummary.kt

```diff
@@ -0,0 +1,103 @@
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material3.*
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.cornellappdev.score.model.ScoreEvent
+import com.cornellappdev.score.theme.Style.bodyNormal
+import com.cornellappdev.score.theme.Style.heading5
+import com.cornellappdev.score.util.scoreEvents1
+
+
+@Composable
+fun ScoringSummary(scoreEvents: List<ScoreEvent>) {
```

Comment:
Please add a 

`modifier: Modifier = Modifier` parameter here and use that as the modifier for the lazy column. It looks like the horizontal padding here is none, which is fine--we just need to make sure people in the future can add padding if needed on the actual screen. Adding a modifier allows this.